dist/generate-custom-parser.js

```diff
@@ -1819,7 +1840,9 @@ var Resource = {
   }(),
   generateDoc: function generateDoc(_ref) {
     var content = _ref.body,
-        response = _ref.response;
+        response = _ref.response,
+        _ref$alreadyDecoded = _ref.alreadyDecoded,
+        alreadyDecoded = _ref$alreadyDecoded === void 0 ? false : _ref$alreadyDecoded;
```

Comment:
since it's much shorter and easier to read, this might become a one-liner: `decoded = (_ref.decoded === void 0) ? false : _ref.decoded`

Reply:
Sounds good! Sorry for the nitpick!